spyder/plugins/editor/widgets/codeeditor.py

```diff
@@ -1159,10 +1158,18 @@ def stop_completion_services(self):
         logger.debug('Stopping completion services for %s' % self.filename)
         self.completions_available = False
 
-    @schedule_request(method=CompletionRequestTypes.DOCUMENT_DID_OPEN,
-                      requires_response=False)
+    @request(method=CompletionRequestTypes.DOCUMENT_DID_OPEN,
+             requires_response=False)
```

Comment:
Very good catch @rear1019! You're right in your commit message that we're not respecting the LSP protocol by sending a `didOpen` after a `didChange`.

Reply:
Please let me know if I should a regression test for this.